ui/shared/utils/constants.js

```diff
@@ -376,6 +376,12 @@ export const CATEGORY_FAMILY_FILTERS = {
         ({ sampleType, datasetType }) => sampleType !== SAMPLE_TYPE_RNA && datasetType === dataType,
       ),
     })),
+    {
+      value: `${SHOW_DATA_LOADED}_PHENO`,
+      name: 'Data Loaded - Phenotype Prioritization',
+      // eslint-disable-next-line no-unused-vars
+      createFilter: (family, _user, _samplesByFamily) => family.hasPhenotypePrioritization,
```

Comment:
javascript does not care if you pass extra positional arguments, so you can just do 
```
createFilter: family => family.hasPhenotypePrioritization,
```